packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-pending-task/ola-pending-task.routing.js

```diff
@@ -0,0 +1,45 @@
+import filter from 'lodash/filter';
+
+const redirectTo = (transition) => {
+  const injector = transition.injector();
+  return injector
+    .getAsync('serverName')
+    .then((serverName) => {
+      return injector
+        .getAsync('DedicatedServerInterfacesService')
+        .then((DedicatedServerInterfacesService) =>
+          DedicatedServerInterfacesService.getTasks(serverName).then(
+            (tasks) => {
+              const pendingTasks = filter(tasks, ({ status }) => {
+                return ['todo', 'init', 'doing'].includes(status);
+              });
+              return pendingTasks;
+            },
+          ),
+        );
+    })
+    .then((pendingTasks) => {
+      return pendingTasks.length > 0
+        ? 'app.dedicated.server.interfaces.ola-pending-task'
+        : null;
+    })
+    .catch(() => null);
+};
+
+const routing = /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated.server.interfaces.ola-pending-task', {
+    url: '/ola-pending-task',
+    translations: { value: ['.'], format: 'json' },
```

Comment:
```suggestion

```
We should prefer the `.run(/* @ngTranslationsInject:json ./translations */)` hook